hlmview/utils.py

```diff
@@ -213,3 +220,34 @@ def get_devices_data(coordinator):
             devices.append(device_data)
     
     return devices
+
+
+def get_object_module(object_id: int, object_class: int = None):
+    if object_class is None:
+        obj = GamObject.get_or_none(GamObject.ob_id == object_id)
+        if obj is None:
+            return None
+        object_class = obj.ob_objecttype.ot_objectclass.oc_name
+    if object_class in [ObjectClassID.VESSEL.value, ObjectClassID.CRYOSTAT.value]:
+        return _get_module_object(object_id, ObjectTypeID.SLD.value)
+    elif object_class == ObjectClassID.GAS_COUNTER.value:
+        return _get_module_object(object_id, ObjectTypeID.GCM.value)
+    else:
+        return None
+
+
+
+def _get_module_object(object_id: int, module_type: int):
```

Comment:
Same comment I had on the HLM import issue, it seems odd to have a comment explaining this private function, rather than the public function calling it.